README.md

```diff
@@ -207,23 +220,273 @@ More information can be found [here](http://www.diveintopython3.net/strings.html
+```r
+?setGeneric
+```
+
+### Expose a subset of functions within a Python module
+
+For many reasons, some Python functions are meant to be used internally, and these do not need to be exposed the R package. For example, `notifyMe` is a Python decorator, and can only be used for other Python functions.
+
+Let's omit the following functions from `synapseutils` module:
+* `copyFileHandles`
+* `notifyMe`
+
+You would need to specify a function like `selectFunctions` below in a shared .R file:
+```r
+toOmit <- c("copyFileHandles", "notifyMe")
+selectFunctions <- function(x) {
+    if (any(x$name == toOmit)) {
+        return(NULL)
+    }
+    x
+}
+```
+
+Then from the R script that generates .Rd files, update `generateRdFiles` as following:
+```r
+generateRdFiles(srcRootDir,
+                pyPkg = "synapseutils",
+                module = "synapseutils",
+                modifyFunctions = selectFunctions)
+```
+
+And in your `.onLoad`, update `generateRWrappers` as following:
+```r
+generateRWrappers(pyPkg = "synapseutils",
+                  module = "synapseutils",
+                  setGenericCallback = callback,
+                  modifyFunctions = selectFunctions)
+```
+
+### Expose a subset of classes within a Python module
+
+The `synapseclient` module in `synapsePythonClient` has the following structure:
+```
+    module: synapseclient
+        module: client
+            class: Synapse
+                function: get
+                function: login
+        module: entity
+            class: Entity
+                function: get
+                function: privateGet
+            class: File
+                function: get
+                function: privateGet
+            class: Folder
+                function: get
+                function: privateGet
+```
+
+Now let's try a more complicated example where we want to expose the following functions and classes from `synapseclient.entity` module:
+```
+  class: File
+      function: get
+  class: Folder
+      function: get
+```
+
+Note that we do not want the following from the `synapseclient.entity` module:
+* `function: privateGet` from any of the classes
+* `class: Entity`
+
+First, define your `selectClasses` in the shared .R file:
+```r
+methodsToOmit <- "privateGet"
+classToSkip <- "Entity"
+selectClasses <- function(class) {
+    if (any(class$name == classToSkip)) {
+        return(NULL)
+    }
+    if (!is.null(class$methods)) {
+        culledMethods <- lapply(X = class$methods, function(x) {
+            if (any(x$name == methodsToOmit)) NULL else x;
+        })
+        # Now remove the nulls
+        nullIndices <- sapply(culledMethods, is.null)
+        if (any(nullIndices)) {
+            class$methods <- culledMethods[-which(nullIndices)]
+        }
+    }
+    class
+}
+```
+
+Then call `generateRdFiles` and `generateRWrappers` as following:
+```r
+generateRdFiles(srcRootDir,
+                pyPkg = "synapseclient",
+                module = "synapseclient.entity",
```

Comment:
In the reference doc' please document 'selectClasses' thoroughly.  What are the inputs and outputs? How does it change what generatedRdFiles does?